cmd/main.go

```diff
@@ -25,6 +24,10 @@ const (
 )
 
 func main() {
+	_, err := maxprocs.Set()
+	if err != nil {
+		return
+	}
```

Comment:
Error should be ignored as this is an optimization. It could be logged as a warning, but shouldn't return.